src/neo/Cryptography/ECC/ECPoint.cs

```diff
@@ -39,6 +48,12 @@ public int CompareTo(ECPoint other)
             return Y.CompareTo(other.Y);
         }
 
+        /// <summary>
+        /// Decode an <see cref="ECPoint"/> object from a sequence of byte.
```

Comment:
Mention ANSI X9.62 4.3.6 and support for both compressed and uncompressed forms?